asset-transfer-sbe/README.md

```diff
@@ -0,0 +1,56 @@
+# Sample to demonstrate State-Based Endorsements (SBE)
+
+## Introduction to SBE
+Fabric allows multiple ways to set endorsements for transactions on the network. The default method is specified in the chaincode definition, which is agreed by channel members and committed to the channel. However, there are cases where it may be necessary for a particular public channel state or private data collection state (particular key) to have a different endorsement policy. State-based endorsement allows endorsement policies to be overridden by a different policy for the specified keys. State-Based Endorsements are also known as Key Level Endorsements. To learn more about endorsement policies and State-Based endorsements, visit the [Fabric Endorsement Policies documentation](https://hyperledger-fabric.readthedocs.io/en/master/endorsement-policies.html#multiple-ways-to-require-endorsement).
+
+
+Fabric Shim Method's for State-Based endorsements (below references from fabric-shim, ChaincodeStub for Node):
+- setStateValidationParameter(key: string, ep: Uint8Array): Promise<void>;
+- getStateValidationParameter(key: string): Promise<Uint8Array>;
+- setPrivateDataValidationParameter(collection: string, key: string, ep: Uint8Array): Promise<void>;
+- getPrivateDataValidationParameter(collection: string, key: string): Promise<Uint8Array>;
+            
+
```

Comment:
I think your last para is a great description of the functionality of this sample. Consider moving it here, for e.g.
```
## About the SBE Asset Transfer Sample
This example demonstrates modifying the default chaincode policy of majority (requiring endorsements from both Org Peers), and changing it to a less restrictive policy requiring endorsement of a specific Org Peer for endorsements of a specified Key. Endorsements can also be modified to make them more restrictive for a particular Key in a similar way.

On Asset creation, endorsement policy for the Asset Key is set to either Org1 or Org2 peer, depending on the client Org creating the Asset. Creation of Asset still needs the default chaincode endorsement policy, and hence will need to be endorsed by both Org's (Org1 & Org2). However, all future updates for the Asset will use the modified endorsements as per the State-based endorsements set during Asset creation for the specified Key, instead of the default chaincode endorsement policy.

Hence transactions like UpdateAsset or TransferAsset will not succeed if it is endorsed by the non-owner Org Peer.
```

Reply:
thanks, this really makes sense to move upwards, done